copyright

```diff
@@ -508,6 +508,19 @@ Copyright: Lineth (https://github.com/Lineth)
 License: CC-BY-SA-4.0
 Comment: Derived from public domain sounds taken from freesound.org.
 
+Files:
+ images/outfit/pug?gridfire?turret*
+ images/hardpoint/pug?gridfire?turret*
+Copyright: Becca Tommaso
+License: CC-BY-SA-3.0
+Comment: Based on 10010101001 (https://github.com/10010101001) concept, derived from works by Michael Zahniser (under the same license) 
+
+Files:
+ images/outfit/T3?anti?missile*
+ images/hardpoint/T3?anti?missile*
+Copyright: Becca Tommaso
+License: CC-BY-SA-3.0
+Comment: Derived from works by Michael Zahniser (under the same license) 
```

Comment:
Is it? Which works then? Most Pug outfit sprites were made Frederick Goy IV (@comnom).

Reply:
Right, i corrected it